src/embedded/types.ts

```diff
@@ -1,12 +1,53 @@
-export interface EmbeddedMessageUpdateHandler {
+export interface IterableEmbeddedButtonAction {
+  type: string;
+  data?: string;
+}
+
+interface IterableEmbeddedElements {
```

Comment:
@mprew97 Did we decide to export all these?

Reply:
yep, miss on my part. done now.